### PR TITLE
docker manager fixes

### PR DESCRIPTION
**Pull request by Jimbo4794**

### CODE REVIEW

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.kubernetes.manager.ivt/.factorypath [hunk 1]
```diff
@@ -0,0 +1,58 @@
+<factorypath>
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.kubernetes.manager.ivt/.factorypath]: Need to remove these files from the repo, add to .gitignore at the base of the repo and delete

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.kubernetes.manager/.factorypath [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<factorypath>
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.kubernetes.manager/.factorypath]: as with the other .factorypath

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager.ivt/src/main/java/dev/galasa/http/manager/ivt/HttpManagerIVT.java [hunk 3]
```diff
@@ -61,18 +61,18 @@ public void makeJsonRequest() throws HttpClientException, URISyntaxException, Ex
...
-        boolean fileExists = false;
-        client3.setURI(new URI("https://p2.galasa.dev"));
-        //client3.setAuthorisation("username", "password");
-        client3.getFile(Paths.get("/tmp/dev.galasa_0.3.0.jar"), "/plugins/dev.galasa_0.3.0.jar");
-        File f = new File("/tmp/dev.galasa_0.3.0.jar");
-        if (f.exists() && !f.isDirectory()) {
-            fileExists = true;
-        }
-        assertThat(fileExists).isTrue();
+    // @Test
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager.ivt/src/main/java/dev/galasa/http/manager/ivt/HttpManagerIVT.java]: why commenting out?

File: galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager.ivt/.factorypath [hunk 4]
```diff
@@ -0,0 +1,59 @@
+<factorypath>
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager.ivt/.factorypath]: as with the other .factorypath

File: galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/.factorypath [hunk 5]
```diff
@@ -0,0 +1,57 @@
+<factorypath>
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-other-parent/dev.galasa.galasaecosystem.manager/.factorypath]: as with the other .factorypath

File: galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/pom.xml [hunk 6]
```diff
@@ -47,7 +47,12 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.6</version>
 		</dependency>
+	
+		<dependency>
```
Comment (Michael-Baylis) [inline galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.docker.manager/pom.xml]: does compress need to be added to docker or http managers?   don't believe it needs to be both.

File: galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager.ivt/src/main/java/dev/galasa/http/manager/ivt/HttpManagerIVT.java (hunk 3 above)
Comment (Jimbo4794) [reply galasa-managers-parent/galasa-managers-comms-parent/dev.galasa.http.manager.ivt/src/main/java/dev/galasa/http/manager/ivt/HttpManagerIVT.java]: Thanks id forgotten id done that, i need to change it work with the new getFile logic

**Pull request merged**